sdcm/nemesis.py

```diff
@@ -1693,12 +1693,8 @@ def choose_partitions_for_delete(self, partitions_amount, ks_cf, with_clustering
                 if exclude_partitions and partition_key in exclude_partitions:
                     continue
 
-                # Get the max cl value in the partition according to the table's clustering order.
-                table_clustering_order = get_table_clustering_order(ks_cf=ks_cf, ck_name='ck', session=session)
-                # Build the query 'from the right side' so it finds the highest cl value using 'limit 1'.
-                # So it has to check the table's clustering order and set a reversed order if needed.
-                reversed_order = 'desc' if table_clustering_order.lower() == 'asc' else 'asc'
-                cmd = f"select ck from {ks_cf} where pk={partition_key} order by ck {reversed_order} limit 1"
+                # Get the max cl value in the partition.
+                cmd = f"select ck from {ks_cf} where pk={partition_key} order by ck desc limit 1"
```

Comment:
is it ordering by value, right?

Reply:
yes, so for example, test now has:
```
< t:2022-02-24 12:50:49,199 f:nemesis.py      l:1722 c:sdcm.nemesis         p:DEBUG > sdcm.nemesis.DeleteRowsMonkey: Partitions for delete: defaultdict(<class 'list'>, {122: [0, 5554], 124: [0, 5554], 126: [0, 5554], 128: [0, 5554], 130: [0, 5554], 132: [0, 5554], 134: [0, 5554], 136: [0, 5554], 138: [0, 5554], 140: [0, 5554]})
```

Reply:
why not just revert to exactly what was there before ? since this nemesis is creating it's own tables, I don't think you need to mess with it. (as the original comment suggests)
```
                # The scylla_bench.test table is created WITH CLUSTERING ORDER BY (ck DESC).
                # So first returned value is max cl value in the partition
                cmd = f"select ck from {ks_cf} where pk={partition_key} limit 1"
```

Reply:
It is the table for s-b and it won't work if table is created with "order by asc".

Reply:
@fruch i think it runs ok so we can merge existing fix.